Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java

```diff
@@ -1352,4 +1357,28 @@ public String getService () {
         key.append(getResources().getString(R.string.service));
         return key.toString();
     }
+
+    public void toggleLoader(final boolean visible) {
+        activity.runOnUiThread(new Runnable() {
```

Comment:
use lambda function here, and loader.setVisibility can be :-

`loader.setVisibility(visible ? View.VISIBLE : View.GONE)`